park_api/geodata.py

```diff
@@ -6,7 +6,7 @@
 from park_api.util import remove_special_chars
 
 
-lot_fields = ['name', 'id', 'type', 'lng', 'lat', 'address', 'total']
+lot_fields = ['name', 'id', 'type', 'lng', 'lat', 'address', 'total', 'aux']
```

Comment:
The aux field can be filled with arbitrary objects to add additional data or mappings.